source/rover.js

```diff
@@ -17,6 +17,15 @@ this.initialize = function() {
     if ( this.id === "source/rover.vwf" ) {
         return;
     }
+
+    //TODO: Set this properly (we shouldn't have to set this explicity in here...
+    // this.transform gets set automatically for ONE of the rovers, but not the other one)
+    this.transform = [
+        1,  0,  0,  0,
+        0,  1,  0,  0,
+        0,  0,  1,  0,
+        0,  0,  0,  1
+    ];
```

Comment:
I'm not sure I understand what's going on here. Why wouldn't the other rovers have a transform? What were you seeing prior to adding this code?

@eric79 Any thoughts? If there is something wrong here, it could be a larger issue.